site/zenodo_rdm/curation/rules.py

```diff
@@ -4,19 +4,22 @@
 #
 # ZenodoRDM is free software; you can redistribute it and/or modify
 # it under the terms of the MIT License; see LICENSE file for more details.
-
 """Rules for curation."""
 
+from datetime import datetime
+
 from flask import current_app
 from invenio_records_resources.proxies import current_service_registry
 
 
 def award_acronym_in_description(record):
     """Check if EU award name in record description."""
     award_service = current_service_registry.get("awards")
-    description = record.metadata["description"]
-    funding = record.metadata["funding"]
+    description = record.metadata.get("description")
+    if not description:
+        return False
 
+    funding = record.metadata["funding"]
     for f in funding:
         if f["funder"]["id"] == "00k4n6c32":
```

Comment:
```suggestion
        if f["funder"].get("id") == "00k4n6c32":
```

minor: again, I know that we prefilter the data in the query, but in case we change the logic at some point it's good to be on the safe side